packages/web/tests/components/Contributions/ContributionsBox.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '../../testUtils/testTools';
+import { ContributionsBox } from '../../../src/components/Contributions';
+import { ContributionsList } from '../../../src/pages/app/contributions';
+// Mock repo objects
+const clist1: ContributionsList = {
+  id: '1',
+  nodeID: 'PR_00000',
+  description: 'clist1',
+  type: 'CLOSED',
+  score: 1,
+  contributedAt: new Date('1612-04-20'),
+  url: 'abc.com',
```

Comment:
Since this is an open source project, we shouldn't be using branded sites:
```suggestion
  url: 'some-fake-url.com',
```